templates/etc/icinga2/constants.conf.j2

```diff
@@ -0,0 +1,32 @@
+/**
+ *  {{ ansible_managed }}
+ */
+
+/**
+ * This file defines global constants which can be used in
+ * the other configuration files.
+ */
+
+/* The directory which contains the plugins from the Monitoring Plugins project. */
+const PluginDir = "/usr/lib64/nagios/plugins"
+
+/* The directory which contains the Manubulon plugins.
+ * Check the documentation, chapter "SNMP Manubulon Plugin Check Commands", for details.
+ */
+const ManubulonPluginDir = "/usr/lib64/nagios/plugins"
+
+/* The directory which you use to store additional plugins which ITL provides user contributed command definitions for.
+ * Check the documentation, chapter "Plugins Contribution", for details.
+ */
+const PluginContribDir = "/usr/lib64/nagios/plugins"
+
+/* Our local instance name. By default this is the server's hostname as returned by `hostname --fqdn`.
+ * This should be the common name from the API certificate.
+ */
+const NodeName = "{{ inventory_hostname }}"
+
+/* Our local zone name. */
+const ZoneName = "{{ inventory_hostname }}"
+
+/* Secret key for remote node tickets */
+const TicketSalt = "{{ icinga2_salt|default("") }}"
```

Comment:
Undefined variable?

Reply:
I'm not yet sure if the TicketSalt must be configured on clients, so the default is empty. I will check and update this if needed after more testing